packages/icons-angular/src/templates-compat.js

```diff
@@ -0,0 +1,196 @@
+      title: this.title,
+      "aria-label": this.ariaLabel,
+      "aria-labelledby": this.ariaLabelledby,
+      "aria-hidden": this.ariaHidden,
+      focusable: this.isFocusable.toString()
+    });
+
+    const attrKeys = Object.keys(attributes);
+    for (let i = 0; i < attrKeys.length; i++) {
+      const key = attrKeys[i];
+      const value = attributes[key];
+      if (key === "title") {
+        continue;
+      }
+      if (value) {
+        svg.setAttribute(key, value);
+      }
+    }
+
+    if (attributes.title) {
+      const title = document.createElement("title");
+      title.textContent = attributes.title;
+      ${className}Directive.titleIdCounter++;
+      title.setAttribute("id", \`${iconName}-$\{${className}Directive.titleIdCounter}\`);
+      svg.appendChild(title);
+      svg.setAttribute("aria-labelledby", \`${iconName}-$\{${className}Directive.titleIdCounter}\`);
+    }
+	}
+}
+
+@NgModule({
+  declarations: [
+    ${className},
+    ${className}Directive
+  ],
+  exports: [
+    ${className},
+    ${className}Directive
+  ]
+})
+export class ${className}Module {}
+`;
+
+const iconStoryTemplate = icon => `.add("${icon.moduleName}", () => ({
+  template: \`
+    <p>Component <code>&lt;ibm-icon-${param(
+      icon.moduleName
+    )}&gt;&lt;/ibm-icon-${param(icon.moduleName)}&gt;</code></p>
+    <ibm-icon-${param(icon.moduleName)}></ibm-icon-${param(icon.moduleName)}>
+    <p>Directive <code>&lt;svg ibmIcon${pascal(
+      icon.moduleName
+    )}&gt;&lt;/svg&gt;</code></p>
+    <svg ibmIcon${pascal(icon.moduleName)}></svg>
+  \`
+}))
+.add("${icon.moduleName} with label", () => ({
+  template: \`
+    <ibm-icon-${param(
+      icon.moduleName
+    )} ariaLabel="label for the icon"></ibm-icon-${param(icon.moduleName)}>
+    <svg ibmIcon${pascal(icon.moduleName)} ariaLabel="label for the icon"></svg>
+  \`
+}))
+.add("${icon.moduleName} with title", () => ({
+  template: \`
+    <ibm-icon-${param(icon.moduleName)} title="icon title"></ibm-icon-${param(
+  icon.moduleName
+)}>
+    <svg ibmIcon${pascal(icon.moduleName)} title="icon title"></svg>
+  \`
+}))
+.add("${icon.moduleName} with class on the SVG", () => ({
+  template: \`
+    <ibm-icon-${param(
+      icon.moduleName
+    )} innerClass="test-class another-class"></ibm-icon-${param(
+  icon.moduleName
+)}>
+    <svg ibmIcon${pascal(
+      icon.moduleName
+    )} class="test-class another-class"></svg>
+  \`
+}))`;
+
+const gerateIconStories = icons => {
+  let value = '';
+  for (const icon of icons) {
+    value += iconStoryTemplate(icon);
+  }
+  return value;
+};
+
+const generateStoryImports = icons => {
+  let imports = '';
+  for (const icon of icons) {
+    imports += `import { ${
+      icon.moduleName
+    }Module } from "./../${icon.outputOptions.file.replace('es', 'lib')}"\n`;
+  }
+  return imports;
+};
+
+const storyTemplate = (basename, icons) => `
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+
+${generateStoryImports(icons)}
```

Comment:
Not sure if this has come up, but feel free to only use one icon in the storybook instead of generating ones for each one!